gnucash/gnome/window-reconcile.c

```diff
@@ -777,6 +778,34 @@ startRecnWindow(GtkWidget *parent, Account *account,
         label = GTK_WIDGET(gtk_builder_get_object (builder, "end_label"));
         gtk_label_set_mnemonic_widget(GTK_LABEL(label), end_value);
 
+        box = GTK_WIDGET(gtk_builder_get_object (builder, "previous_dates"));
+
+        reconcile_dates = xaccAccountGetReconcileStatementDates(account);
+
+        if (reconcile_dates)
+        {
+            PWARN ("good");
```

Comment:
For future sanity protection please expand that to say what's good. Does it really need to be a warning that shows up in the trace file every time? Seems an `PINFO` (emitted when user specifies `--debug`) or a `DEBUG` ( when user specifies `--log gnc.gui=debug`) would be more appropriate.

Reply:
This was one of many silly messages to trace while I'm figuring out swig. removed!